.circleci/config.yml

```diff
@@ -38,6 +38,33 @@ aliases:
       keys:
         - v1-build-{{ .Revision }}
         - v1-build-
+  - &build_pr
+    <<: *defaults
+    resource_class: medium+
+    steps:
```

Comment:
If you move the `store_test_results` in `build` to be before the `save_*_cache` steps, then these `build_pr` steps are identical to the middle steps of `build`. Is there a way to define them once and then include them in both places, like with the `&` / `*` / `<<:` syntax or something similar?

Another way to handle this could be conditional steps, where it would skip certain steps if running in a context where we don't want them. I do this when building the Mac & Windows app: I have a `should_sign` alias which checks the current branch to determine whether or not we want to sign the current build, and I added `condition: *should_sign` to steps which should only run for signed builds.

Reply:
To make this work we might want to remove restoring the git_cache.  We should do that as a next step.